doc/source/glossary.rst

```diff
@@ -95,54 +99,62 @@ Glossary
 
 
    along an axis
-       Axes are defined for arrays with more than one dimension.  A
-       2-dimensional array has two corresponding axes: the first running
-       vertically downwards across rows (axis 0), and the second running
-       horizontally across columns (axis 1).
-
-       Many operations can take place along one of these axes.  For example,
-       we can sum each row of an array, in which case we operate along
-       columns, or axis 1::
-
-         >>> x = np.arange(12).reshape((3,4))
-
-         >>> x
-         array([[ 0,  1,  2,  3],
-                [ 4,  5,  6,  7],
-                [ 8,  9, 10, 11]])
+       Reversing along axis 0 (the row axis) below reverses the columns, and
+       reversing along axis 1 (the column axis) reverses the rows. This might
+       be the opposite of what a new user would expect.
 
-         >>> x.sum(axis=1)
-         array([ 6, 22, 38])
+           >>> a = np.arange(12).reshape(3,4)
+           >>> a
+           array([[ 0,  1,  2,  3],
+                  [ 4,  5,  6,  7],
+                  [ 8,  9, 10, 11]])
+           <BLANKLINE>
```

Comment:
This looks like a misuse of blankline - numpy does not emit a blank line here.

Reply:
It might be misleading if the text were part of a docstring, but here it's used for formatting the documentation and is invisible to the reader.

Reply:
Sorry, missed your point initially. Agreed, will change.

Reply:
Removed in 511d769.